### PR TITLE
SHS-5093: A11y Level A: Element IDs are not unique, on search results page

### PR DESCRIPTION
**Pull request by hectorlj**
# READY FOR REVIEW

## Summary
- Create unique id for each instance of the default search 

## Steps to Test
Do the following for several sites:
1. Navigate to `/search`
2. Verify search form element ids are unique

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)

### CODE REVIEW

File: docroot/modules/humsci/hs_blocks/hs_blocks.module [hunk 1]
```diff
@@ -116,3 +116,16 @@ function hs_blocks_preprocess_block__layout_builder(&$variables) {
     $variables['label'] = $variables['configuration']['label'];
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function hs_blocks_preprocess_block__views_exposed_filter_block(&$variables) {
+  if (str_contains($variables['derivative_plugin_id'], "search-search_page")) {
```
Comment (cienvaras) [inline docroot/modules/humsci/hs_blocks/hs_blocks.module]: Please change the double quotes to single quotes:

```suggestion
  if (str_contains($variables['derivative_plugin_id'], 'search-search_page')) {
```

File: docroot/modules/humsci/hs_blocks/hs_blocks.module [hunk 2]
```diff
@@ -116,3 +116,16 @@ function hs_blocks_preprocess_block__layout_builder(&$variables) {
...
+    // Append the build id to create
+    // unique form id.
+    $build_id = $variables['content']['#build_id'];
+    $variables['content']['#id'] .= '-' . $build_id;
+    $variables['#cache']['max-age'] = 0;
```
Comment (cienvaras) [inline docroot/modules/humsci/hs_blocks/hs_blocks.module]: Are you sure this is needed? I did a quick test with the cache enabled and I still got different ids for both blocks. I might be doing something wrong in my test, but please check to confirm if it's needed.

**Review by cienvaras — CHANGES_REQUESTED** [review]
Works as expected, but I have a question and also found a minor thing to fix. Please see inline comments.

File: docroot/modules/humsci/hs_blocks/hs_blocks.module (hunk 2 above)
Comment (hectorlj) [reply docroot/modules/humsci/hs_blocks/hs_blocks.module]: True, even if they get cached the ids will still be unique. You're right, it's not necessary to update the id every time. I'll remove it 👍🏼

**Review by cienvaras — APPROVED** [review]
Works great, thanks!

**Pull request merged**